app/jobs/request_otp_sms_job.rb

```diff
@@ -1,11 +1,13 @@
 class RequestOtpSmsJob < ApplicationJob
+  include PatientPhoneNumberHelper
```

Comment:
Since we are already including this on the phone auth model, could we rely on that and not mix in a model specific behavior here into a job? I suppose you could do something like


```ruby
    TwilioApiService.new.send_sms(recipient_number: user.localized_phone_number), message: otp_message(user), context: context)
```

then

```ruby
class PatientPhoneNumber
  def localized_phone_number
    number_with_country_code(phone_number)
  end
...
```
    
Then the phone number auth is telling the job the correct number to use, and the job can concern itself just with the job handling.

Reply:
Okay. Will make these changes